packages/cache/src/CacheStore.spec.tsx

```diff
@@ -32,98 +32,121 @@ describe('CacheStore', () => {
 
-  it("have clearError method with key should clear promise & error for key's cached", async () => {
-    expect(cacheStore.getError(errorCache(1))).toBeUndefined()
-    expect(cacheStore.getError(errorCache(2))).toBeUndefined()
-    try {
-      cacheStore.suspend(errorCache(1))
-    } catch (promiseToSuspense) {
-      expect(await promiseToSuspense).toBeUndefined()
-    }
-    try {
-      cacheStore.suspend(errorCache(1))
-    } catch (error) {
-      expect(error).toBe(ERROR_MESSAGE)
-    }
-    try {
-      cacheStore.suspend(errorCache(2))
-    } catch (promiseToSuspense) {
-      expect(await promiseToSuspense).toBeUndefined()
-    }
-    try {
-      cacheStore.suspend(errorCache(2))
-    } catch (error) {
-      expect(error).toBe(ERROR_MESSAGE)
-    }
-    expect(cacheStore.getError(errorCache(1))).toBe(ERROR_MESSAGE)
-    expect(cacheStore.getError(errorCache(2))).toBe(ERROR_MESSAGE)
+      cacheStore.clearError(errorCache(1))
+      expect(cacheStore.getError(errorCache(1))).toBeUndefined()
+      expect(cacheStore.getError(errorCache(2))).toBe(ERROR_MESSAGE)
+      cacheStore.clearError(errorCache(2))
+      expect(cacheStore.getError(errorCache(1))).toBeUndefined()
+      expect(cacheStore.getError(errorCache(2))).toBeUndefined()
+    })
+  })
 
-    cacheStore.clearError(errorCache(1))
-    expect(cacheStore.getError(errorCache(1))).toBeUndefined()
-    expect(cacheStore.getError(errorCache(2))).toBe(ERROR_MESSAGE)
-    cacheStore.clearError(errorCache(2))
-    expect(cacheStore.getError(errorCache(1))).toBeUndefined()
-    expect(cacheStore.getError(errorCache(2))).toBeUndefined()
+  describe('remove', () => {
+    it('should take a key and remove the cached data for that key', async () => {
+      try {
+        cacheStore.suspend(successCache(1))
+      } catch (promiseToSuspense) {
+        await promiseToSuspense
+      }
+      expect(cacheStore.getData(successCache(1))).toBe(TEXT)
+      expect(() => {
+        cacheStore.remove(successCache(1))
+      }).not.throw()
+      expect(cacheStore.getData(successCache(1))).toBeUndefined()
+    })
   })
 
-  it('should take a key and remove the cached data for that key', async () => {
-    try {
-      cacheStore.suspend(successCache(1))
-    } catch (promiseToSuspense) {
-      await promiseToSuspense
-    }
-    expect(cacheStore.getData(successCache(1))).toBe(TEXT)
-    expect(() => {
-      cacheStore.remove(successCache(1))
-    }).not.throw()
-    expect(cacheStore.getData(successCache(1))).toBeUndefined()
+  describe('reset', () => {
+    it('should reset cache', async () => {
+      try {
+        cacheStore.suspend(successCache(1))
+      } catch (promiseToSuspense) {
+        await promiseToSuspense
+      }
+      let cached = cacheStore.getData(successCache(1))
+      expect(cached).toBe(TEXT)
+      cacheStore.reset(successCache(1))
+      cached = cacheStore.getData(successCache(1))
+      expect(cached).toBeUndefined()
```

Comment:
We should differ cached.data and cached

```suggestion
      let cachedData = cacheStore.getData(successCache(1))
      expect(cachedData).toBe(TEXT)
      cacheStore.reset(successCache(1))
      cachedData = cacheStore.getData(successCache(1))
      expect(cachedData).toBeUndefined()
```